db/migrate/20170603022915_adds_issued_at_to_donation.rb

```diff
@@ -0,0 +1,14 @@
+class AddsIssuedAtToDonation < ActiveRecord::Migration[5.0]
+  # doin this old-school because we need to initialize it programmatically
+  def up
+  	add_column :donations, :issued_at, :datetime
+  	Donation.all.each do |d|
+  		d.issued_at = d.created_at
+  		d.save
+  	end
```

Comment:
Doing this in a migration has often proved to be painful to me as the migration ages and the project grows. This is probably safe enough for now, but maybe we could consider one of these in the future:

[Rails migrations like a boss](http://railsguides.net/change-data-in-migrations-like-a-boss/)
[Migrations as rake tasks](https://robots.thoughtbot.com/data-migrations-in-rails)
[Seed migrations](http://engineering.harrys.com/2014/06/09/seed-migrations.html)